### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by ViktoriiaMarkus**
- [DEMO LINK](https://ViktoriiaMarkus.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://ViktoriiaMarkus.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form method="post"
+    action="index.html"
+    class="form"
+    >
```
Comment (xiimiik) [inline src/index.html:17]: ❗ checklist point number '5 ❗

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,35 @@
...
+      <input
+      class="big__form"
```
Comment (xiimiik) [inline src/index.html:19]: lets change classname to `input`, and for small input add modifire `--small`
```suggestion
      class="input"
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  border: none;
+
+  box-shadow: 0 1px 8px 0 rgba(0, 0, 0, 0.25);
+  text-shadow: 1px 2px 2px rgba(0, 0, 0, 0.2);
+
+  background: url(images/Search.svg) 15px no-repeat;
+  background-size: 11px;
+  padding-left: 32px;
+  border-radius: 4px;
+}
```
Comment (xiimiik) [inline src/style.css:67]: ❗ ❗ checklist point number '2 ❗ ❗

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+}
```
Comment (xiimiik) [inline src/style.css:14]: checklist point number '7

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  background: url(images/Search.svg) no-repeat, linear-gradient(to top, #f0f0f0, #fff);
+  background-position: 25px;
+}
+
+.big__form:focus::placeholder {
+  font-weight: bold;
+  color: #3d4e61;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  font-size: 16px;
+}
```
Comment (xiimiik) [inline src/style.css:50]: checklist point number '12

**Review by xiimiik — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 6]
```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    method="post"
+    action="index.html"
+    class="form"
+    >
+      <input
```
Comment (denys-danyliuk) [inline src/index.html:19]: add name attributes to inputs

File: src/index.html [hunk 7]
```diff
@@ -11,6 +11,34 @@
...
+      data-qa="big"
+      >
+      <label class="label__big" for="search"></label>
+    </form>
+
+    <form
+    method="post"
+    action="index.html"
+    class="form"
+    >
```
Comment (denys-danyliuk) [inline src/index.html:33]: 5'th point in checklist

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-size: 800px;
+}
+
+.input {
```
Comment (denys-danyliuk) [inline src/style.css:19]: add box-sizing: border box, now your inputs outside of the screen
Знімок екрана 2023-08-01 о 11 28 16
Знімок екрана 2023-08-01 о 11 28 38

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 9]
```diff
@@ -11,6 +11,36 @@
...
+      <input
+      class="input input--small"
+      type="text"
+      name="search__small"
+      id="search-second"
+      placeholder="Los Angeles"
+      data-qa="small"
```
Comment (denys-danyliuk) [inline src/index.html:41]: 5'th point in checklist

File: src/index.html [hunk 10]
```diff
@@ -11,6 +11,36 @@
...
+
+    <form
+    method="post"
+    action="index.html"
+    class="form"
+    >
+      <input
+      class="input input--small"
+      type="text"
+      name="search__small"
```
Comment (denys-danyliuk) [inline src/index.html:38]: 6'th point in checklist

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,70 @@
+.body {
+  margin: 0;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-size: 300;
```
Comment (denys-danyliuk) [inline src/style.css:9]:
```suggestion
  font-weight: 300;
```

File: src/style.css [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-size: 800px;
```
Comment (denys-danyliuk) [inline src/style.css:16]:
```suggestion
  font-weight: 800;
```

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**